tick/eval/eval_number_node.go

```diff
@@ -0,0 +1,62 @@
+package eval
+
+import (
+	"errors"
+	"regexp"
+
+	"github.com/influxdata/kapacitor/tick"
+)
+
+type EvalNumberNode struct {
+	Node *tick.NumberNode
+
+	Type ValueType
+}
+
+func CreateEvalNumberNode(numberNode *tick.NumberNode) NodeEvaluator {
+	return &EvalNumberNode{
```

Comment:
These CreateEval{}Node functions should return their concrete type so if you ever need to create an explicit *EvalNumberNode you can. Then later if it is used as a NodeEvaluator then it will just work.

Also any reason not to use the more standard naming `NewEvalNumberNode` over `CreateEvalNumberNode`?